common/log/log.go

```diff
@@ -118,7 +118,11 @@ func (l *Logger) SetDebugLevel(level int) error {
 func (l *Logger) output(level int, s string) error {
 	gid := GetGID()
 	gidStr := strconv.FormatUint(gid, 10)
-
+	//check the log filesize
+	if CheckIfNeedNewFile() {
+		ClosePrintLog()
```

Comment:
I have test the out function with 100K loop and without print to the stdout.The total time is only increase about 0.06~0.12sec, so i think the performance effect is minimum.
my test case attached, the print to stdout part is commented while testing.
```
func TestOutput(t *testing.T) {
	CreatePrintLog("./")
	for i := 1; i < 100000; i++ {
		Info("Fatal testing")
	}
}
```